src/app-layout/__integ__/toolbar-tooltips/app-layout-toolbar-drawer-trigger-tooltip.test.ts

```diff
@@ -0,0 +1,165 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+import createWrapper from '../../../../lib/components/test-utils/selectors/index';
+import { drawerIds as drawerIdObj } from '../../../../lib/dev-pages/pages/app-layout/utils/drawer-ids';
+import { VISIBLE_MOBILE_TOOLBAR_TRIGGERS_LIMIT } from '../constants';
+import { setupTest } from '../utils';
+
+const wrapper = createWrapper().findAppLayout();
+const drawerIds = Object.values(drawerIdObj);
+
+describe('refresh-toolbar', () => {
+  //using a theme variable below that will be set when iterating over ['refresh-toolbar', 'refresh']
+  const theme = 'refresh-toolbar';
+  const mobileDrawerTriggerIds = drawerIds.slice(
+    0,
+    VISIBLE_MOBILE_TOOLBAR_TRIGGERS_LIMIT + (theme === 'refresh-toolbar' ? 1 : 0)
+  );
+
+  describe.each(['desktop', 'mobile'] as const)('%s', size => {
+    const drawerIdsToTest = size === 'mobile' ? mobileDrawerTriggerIds : drawerIds; //toolbarDrawerIds;
+    const firstDrawerTriggerSelector = wrapper.findDrawerTriggerById(drawerIdsToTest[0]).toSelector();
+    const expectedFirstDrawerTriggerTooltipText = 'Security'; //must match drawerItems[0].ariaLabels.drawer from /lib/dev-pages/pages/app-layout/utils/drawers
+    const secondDrawerTriggerSelector = wrapper.findDrawerTriggerById(drawerIdsToTest[1]).toSelector();
+    const expectedSecondDrawerTriggerTooltipText = 'Pro help'; //must match drawerItems[0].ariaLabels.drawer from /lib/dev-pages/pages/app-layout/utils/drawers
+    const triggerTooltipSelector = wrapper.findDrawerTriggerTooltip().toSelector();
+
+    test(
+      'Shows tooltip correctly on drawer trigger for mouse interactions drawer triggers',
+      setupTest({ theme, size }, async page => {
+        await expect(page.getElementsCount(triggerTooltipSelector)).resolves.toBe(0);
+        await page.hoverElement(firstDrawerTriggerSelector);
+        await expect(page.getText(triggerTooltipSelector)).resolves.toBe(expectedFirstDrawerTriggerTooltipText);
+        await expect(page.getElementsCount(triggerTooltipSelector)).resolves.toBe(1);
+        await page.hoverElement(wrapper.findNavigationToggle().toSelector());
+        await expect(page.isExisting(triggerTooltipSelector)).resolves.toBe(false);
```

Comment:
Here is an inconsistency in your testing conditions. The initial check does the following
`await expect(page.getElementsCount(triggerTooltipSelector)).resolves.toBe(0);`

whereas this one checks for false. Let's align these assertions. Same for all tests below